src/main/java/com/laserfiche/api/client/model/AccessKey.java

```diff
@@ -39,4 +43,17 @@ public JWK getJwk() {
     public void setJwk(JWK jwk) {
         this.jwk = jwk;
     }
+
+    public static AccessKey CreateFromBase64EncodedAccessKey(String base64EncodedAccessKey) {
+        String accessKeyStr = decodeBase64(base64EncodedAccessKey);
+        accessKeyStr = accessKeyStr.replace("\\\"", "\"");
+        Gson gson = new GsonBuilder().registerTypeAdapter(JWK.class, new JwkDeserializer()).create();
```

Comment:
We could probably move its initialization out of the method since it will be initialized once and called multiple times.

Reply:
ok done, and if there are not going to be any more changes, then i will publish preview version and update the other 2 java projects